src/main/java/DukeBot/Ui.java

```diff
@@ -0,0 +1,96 @@
+package DukeBot;
+
+import java.util.Scanner;
+
+/**
+ * Encapsulates the User Interface.
+ */
+public class Ui {
+
+    private Scanner scanner;
+
+    /**
+     * Initialises the Ui object and a Scanner that reads the user's input.
+     */
+    public Ui() {
+        this.scanner = new Scanner(System.in);
+    }
+
+    /**
+     * Prints a line onto the output.
+     */
+    public void showLine() {
+        System.out.println("-----------------------------------------------");
+    }
+
+    /**
+     * Prints the new task onto the output.
+     *
+     * @param newTask to be printed on the output.
+     */
+    public void showNewTask(Task newTask) {
+        System.out.println("    Got it. I've added this task:");
```

Comment:
```suggestion
        System.out.println("\tGot it. I've added this task:");
```
Could use `\t` to format tab in front of the phrase you are printing.